packages/components/table/src/thead.tsx

```diff
@@ -15,8 +15,19 @@ import type { SortDirection, Column } from "./use-table"
 import { useTableContext, render } from "./use-table"
 
 export type TableHeadProps = NativeTableHeadProps
+export type AriaSort = {
+  none: "none"
+  asc: "ascending"
+  desc: "descending"
+}
 
 export const Thead = ({ ...rest }: TableHeadProps) => {
+  const ariaSort: AriaSort = {
```

Comment:
Please delete `ariaSort`.

It is also not necessary to define it within the function component. If you do, define it outside the function component.